wwwapp/views.py

```diff
@@ -379,6 +380,49 @@ def participants_view(request, year):
 
     return render(request, 'participants.html', context)
 
+@login_required()
+@permission_required('wwwapp.see_all_workshops', raise_exception=True)
+def lecturers_view(request: HttpRequest, year: int) -> HttpResponse:
+    year = int(year)
+
+    workshops = Workshop.objects.filter(type__year=year).prefetch_related('lecturer', 'lecturer__user')
+
+    people: Dict[int, Dict[str, any]] = {}
+    for workshop in workshops:
+        for lecturer in workshop.lecturer.all():
+            if lecturer.id in people:
+                people[lecturer.id]['workshops'].append(workshop.info_for_client_link())
+                continue
+
+            birth = lecturer.user_info.get_birth_date()
+            is_adult = None
+            if birth is not None:
+                is_adult = settings.WORKSHOPS_START_DATE >= birth + relativedelta(years=18)
```

Comment:
Is this reaaalllyyy neeeded for lecturers? :stuck_out_tongue:
Not sure, @cytadela8 and others what do you think?

Reply:
Why not?

Reply:
It could be useful, but I don't think it's as important for lecturers to have it as one of the few fields visible by default